myRental/modules/database.js

```diff
@@ -3,33 +3,18 @@ var method = dataBaseModule.prototype;
 var fs = require('fs');
 var mysql = require("mysql");
 var bcrypt = require('bcryptjs');
-var knex = require('knex')({
-  client: 'mysql',
-  connection: {
-    host     : '',
-    user     : 'shane',
-    password : 'devPassword',
-    database : 'MY_RENTAL',
-    charset  : 'utf8'
-  }
-});
 
 var User = null;
 
-function dataBaseModule() {
-
-    bookshelf = require('bookshelf')(knex);
-
-    bookshelf.plugin(require('bookshelf-check-duplicates'));
-
-    User = bookshelf.Model.extend ({
-        tableName: 'USERS',
-        duplicates: ['NAME', 'EMAIL'],
-    });
+function dataBaseModule(type) {
+    if(type == "test") {
```

Comment:
Is there any way we can get around having to test for if this is a test inside of production code?

Reply:
I couldn't think of any cleaner way of doing it. I could think over it again before I merge it. If I change it here it might mean the config files are harder to refactor else ware. It may just have to be technical debt to assume.

Reply:
Possibly. Give it a little time, say 20 minutes, to find a better solution. If you can't get anywhere after that, we can deal with this, and in the future we can address it if we have the time.